server/server.js

```diff
@@ -20,16 +19,12 @@ app.use(express.json());
 app.use(express.static(path.join(__dirname, '..', 'build')));
 app.use(express.static('public'));
 
-// Sample GET route
-app.get('/api/data', (req, res) => {
-  const queryString = 'SELECT * FROM users';
-  db.query(queryString).then((data) => res.json({
-    message: 'Seems to work!',
-    data: data.rows,
-  }));
-});
+// Bring in External Routes
+const { searchRoute, newList } = require('./routes/index');
 
+// External Routes
 app.use('/api/search', searchRoute());
+app.use('/api/lists/new', newList());
```

Comment:
Probably going to want to refactor the list routes file to accept multiple CRUD operations like new and edit. So perhaps the route could just be `/api/lists` and the `newList` could contain `router.put('/new',...)` ?